grow/commands/build.py

```diff
@@ -23,7 +25,10 @@
 @click.option('--locate-untranslated',
               default=False, is_flag=True,
               help='Shows untranslated message information.')
-def build(pod_path, out_dir, preprocess, clear_cache, pod_paths, locate_untranslated):
+@click.option('--profile',
```

Comment:
Might be good to move this to a global flag so we can get profiles not on just `grow build` but other operations. For example, `grow stats` takes a while for big sites.